charts/newrelic-infrastructure/README.md

```diff
@@ -1,29 +1,120 @@
+file, the value there is `nil`. If any value is set there, the `lowDataMode` toggle is ignored as this value takes precedence.
+
+Setting this interval above 40 seconds can make you experience issues with the Kubernetes Cluster Explorer so this chart limits setting the interval
+inside the range of 10 to 40 seconds.
+
+### Affinities and tolerations
+
+The New Relic common library allows to set affinities, tolerations, and node selectors globally using e.g. `.global.affinity` to ease the configuration
+when you use this chart using `nri-bundle`. This chart has an extra level of granularity to the components that it deploys:
+control plane, ksm, and kubelet.
+
+Take this snippet as an example:
+```yaml
+global:
+  affinity: {}
+affinity: {}
+
+kubelet:
+  affinity: {}
+ksm:
+  affinity: {}
+controlPlane:
+  affinity: {}
+```
+
+The order to set an affinity is to set first any `kubelet.affinity`, `ksm.affinity`, or `controlPlane.affinity`. If they are empty it fallbacks to
+`affinity` and if it is empty it fallbacks to `global.affinity`.
+
+The same procedure applies to `nodeSelector` and `tolerations`.
+
+On the other hand, some components have affinities and tolerations predefined e.g. to be able to run kubelet pods on nodes that are tainted as master
+nodes or to schedule the KSM scraper on the same node of KSM to reduce the inter-node traffic.
+
+If you are having problems assigning pods to nodes it may be because of this. Take a look at the [`values.yaml`](values.yaml) to see if the pod that is
+not having your expected behavior has any predefined value.
+
+### `hostNetwork` toggle
+
+In versions below v3, changing the `privileged` mode affected the `hostNetwork`. We changed this behavior and now you can set pods to use `hostNetwork`
+using the corresponding [flags from the common library](https://github.com/newrelic/helm-charts/blob/master/library/common-library/README.md)
+(`.global.hostNetwork` and `.hostNetwork`) but the component that scrapes data from the control plane has always set `hostNetwork` enabled by default
+(Look in the [`values.yaml`](values.yaml) for `controlPlane.hostNetwork: true`)
+
+This is because the most common configuration of the control plane components is to be configured to listen only to `localhost`.
+
+If your cluster security policy does not allow to use `hostNetwork`, you can disable it control plane monitoring by setting `controlPlane.enabled` to
+`false.`
+
+### `privileged` toggle
+
+The default value for `privileged` [from the common library](https://github.com/newrelic/helm-charts/blob/master/library/common-library/README.md) is
+`false` but in this particular this chart it is set to `true` (Look in the [`values.yaml`](values.yaml) for `privileged: true`)
 
-## Requirements
+This is because when `kubelet` pods need to run in privileged mode to fetch cpu, memory, process, and network metrics for your nodes.
```

Comment:
```suggestion
This is because `kubelet` pods need to run in privileged mode to fetch cpu, memory, process, and network metrics of your nodes.
```

Reply:
Solved in [#85e36f5](https://github.com/newrelic/nri-kubernetes/pull/409/commits/85e36f5115754dce005f8970e931c32630797de5)